src/shapes/group.class.js

```diff
@@ -531,16 +532,19 @@
      * @return {String} svg representation of an instance
      */
     toSVG: function(reviver) {
-      var markup = [
-        //jscs:disable validateIndentation
-        '<g ',
-          'transform="', this.getSvgTransform(),
+      var markup = this._createBaseSVGMarkup();
```

Comment:
Since we're using `createBaseSVGMarkup` everywhere, perhaps we should just do it in `fabric.Object#toSVG` and then inherit method via `callSuper`.